docs/unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md

```diff
@@ -405,3 +405,142 @@ Algorithm       Hash
 ---------       ----                                                              ----
 SHA256          F46597C0C63A1EEFB200D40EDF654E52F10C3D5D21565886AD603FABAF8D39FB  E:\test_file.txt
 ```
+
+## USB004.001 Enable USB stack (Firmware)
+
+**Test description**
+
+This test aims to verify if USB stack can be enabled. USB stack controls loading
+of UEFI USB drivers.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `SETUP_MENU_KEY` = F2
```

Comment:
Are we sure, that this setup key will be correct for all Dasharo platforms?

Reply:
Any setup isn't valid for all the platforms. Should I remove that part?